### PR TITLE
Fix timestamp to be actual ns

### PR DESCRIPTION
**Pull request by asdil12**
https://progress.opensuse.org/issues/99741

### CODE REVIEW

File: lib/OpenQA/WebAPI/Controller/Admin/Influxdb.pm [hunk 1]
```diff
@@ -28,7 +28,7 @@ sub _queue_output_measure ($url, $key, $tag, $states, $timestamp = undef) {
     }
     $line .= " ";
     $line .= join(',', map { "$_=$states->{$_}i" } sort keys %$states);
-    $line .= ' ' . $timestamp->epoch() * 10e6 if defined $timestamp;
+    $line .= ' ' . $timestamp->epoch() * 10e8 if defined $timestamp;
```
Comment (perlpunk) [inline lib/OpenQA/WebAPI/Controller/Admin/Influxdb.pm:31]:
```suggestion
    $line .= ' ' . $timestamp->epoch() * 1e9 if defined $timestamp;
```

**Review by Martchus — APPROVED** [review]
Having a test for this would be nice, though.

File: lib/OpenQA/WebAPI/Controller/Admin/Influxdb.pm [hunk 2]
```diff
@@ -28,7 +28,7 @@ sub _queue_output_measure ($url, $key, $tag, $states, $timestamp = undef) {
     }
     $line .= " ";
     $line .= join(',', map { "$_=$states->{$_}i" } sort keys %$states);
-    $line .= ' ' . $timestamp->epoch() * 10e6 if defined $timestamp;
+    $line .= ' ' . $timestamp->epoch() * 1e9 if defined $timestamp;
```
Comment (kalikiana) [inline lib/OpenQA/WebAPI/Controller/Admin/Influxdb.pm:31]: Please extend t/api/13-influxdb.t to cover this change

File: t/api/13-influxdb.t [hunk 3]
```diff
@@ -57,16 +57,19 @@ $t->get_ok('/admin/influxdb/minion')->status_is(200)
...
+$rc_fail_finished->subtract(minutes => 3);
 my $sth = $dbh->prepare(
 q!INSERT INTO minion_jobs (id, args, created, delayed, finished, priority, result, retried, retries, started, state, task, worker, queue, attempts, parents, notes)
 	VALUES (7291599, '["/bin/true", 11201356, {"delay": 60, "retries": 1440, "skip_rc": 142, "timeout": "10m", "kill_timeout": "10s"}]', '2023-05-26 17:00:50.542916+02', '2023-05-26 17:00:50.542916+02', ?, 0, null, null, 0, '2023-05-26 17:00:50.565839+02', 'finished', 'hook_script', 1388, 'default', 1, '{}', '{"hook_rc": -1, "hook_cmd": "foobar", "hook_result": "Job is '':investigate:'' already, skipping investigation\n"}')!
 );
 $sth->execute($rc_fail_finished);
 $t->get_ok('/admin/influxdb/minion')->status_is(200)
   ->content_like(qr!openqa_minion_jobs,url=http://example.com active=0i,delayed=1i,failed=0i,inactive=2i!)
-  ->content_like(qr!openqa_minion_jobs_hook_rc_failed,url=http://example.com rc_failed_per_10min=1i \d+!)
+  ->content_like(qr!openqa_minion_jobs_hook_rc_failed,url=http://example.com rc_failed_per_10min=1i $rc_fail_timespan_end_ns!)
```
Comment (perlpunk) [inline t/api/13-influxdb.t:72]: I think would actually just check for the right amount of digits (19). That should work for quite a while :)

**Review by okurz — APPROVED** [review]

**Pull request merged**